src/main/java/org/openhab/binding/zwave/internal/protocol/serialmessage/AddNodeMessageClass.java

```diff
@@ -77,16 +77,20 @@ public ZWaveSerialPayload doRequestStart(boolean highPower, boolean networkWide)
     }
```

Comment:
It is totally related because the message without a callback eats up 4 seconds of the initialization of battery devices before it is cancelled.  It is really critical to the "One and Done" idea.  I implore you to reconsider, no I beg you.  We really would need to raise the default cap to 10 seconds to counteract leaving this in.

Reply:
This should not be required, and it is not related to the management of the wakeup queue so I really would not like it hidden away in here.

I'm still to be convinced it is needed at all as the controller ought to respond in a timely fashion, but it is not part of the wakeup queue management in any case.

Reply:
I'll remove these changes, but it makes me sad because unless you agree to raise the default maxAwakePeriod to 10 seconds from 5 seconds there will not even be a chance for "One and Done" initialization of battery devices at the default settings.  It also raises the importance of the delayed queueKick() at 2 seconds
[batteryfeedbackchanges.txt](https://github.com/openhab/org.openhab.binding.zwave/files/8611674/batteryfeedbackchanges.txt)
 to get initialization going again.  What I see in this file is 1) the 4 second delay caused by this doomed message (callback=0) 2) The kickQueue() kicking in at 2 seconds on the next awake 3) the whole process getting out of whack with a message timeout, but eventually getting done.  Basically the current situation, but improved a bit with the kickQueue

Also related to a comment below.  Node 3 was done after I changed the parameter in the UI to 10 seconds.  It looked like it was changed from the API/JSON, but was not.   Otherwise the same observations for a second time.